src/components/useraccount/PersonalDetails.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="personal-details col-sm-9" v-if="me">
+          <div class="personal-details-text-one">
+            <span>{{ $t('welcomeBack', { name: me.customer.firstName }) }}</span>
+          </div>
+          <div class="personal-details-text-two">
+            {{ $t('welcomeDescription') }}
+            <br>
+            <span class="customer-number">{{ me.customer.customerNumber }}</span>
+          </div>
+          <div class="personal-details-title">
+              <span>{{ $t('title') }}</span>
+              <span v-if="showEditForm"
+                    class="pull-right required-text personal-details-edit-show">
+                {{ $t('main.form.required') }}*
+              </span>
+          </div>
+          <transition name="fade"
+                      mode="out-in">
+            <EditProfileForm v-if="showEditForm"
+                             @close="showEditForm = false"/>
+            <div v-else
+                 class="personal-details-edit-hide">
+              <div class="personal-details-box">
+                <div>
+                  <span data-test="user-profile-name">{{ me.customer.firstName }} {{ me.customer.lastName}}</span>
+                </div>
+                <div data-test="user-profile-email">{{ me.customer.email }}</div>
+                <br>
+                <!--{{#if content.customerInfo.subscribed}}-->
+                <!--<div>{{ $t('subscribedToNewsletter') }}</div>-->
+                <!--{{/if}}-->
+                <div class="personal-details-box-edit">
+                  <button @click="showEditForm = true"
```

Comment:
in Sunrise we usually apply a method for the `@click`  event instead of setting its value in the template.